hloc/extractors/r2d2.py

```diff
@@ -0,0 +1,55 @@
+import sys
+from pathlib import Path
+import torch
+import torchvision.transforms as tvf
+
+from ..utils.base_model import BaseModel
+
+r2d2_path = Path(__file__).parent / "../../third_party/r2d2"
+sys.path.append(str(r2d2_path))
+from extract import load_network, NonMaxSuppression, extract_multiscale
+
+RGB_mean = torch.cuda.FloatTensor([0.485, 0.456, 0.406])
+RGB_std  = torch.cuda.FloatTensor([0.229, 0.224, 0.225])
+norm_RGB = tvf.Normalize(mean=RGB_mean, std=RGB_std)
+
+class R2D2(BaseModel):
+    default_conf = {
+        'checkpoint_name': 'r2d2_WASF_N16.pt',
+        'top-k': 5000,
+
+        'scale-f': 2**0.25,
+        'min-size': 256,
+        'max-size': 1024,
+        'min-scale': 0,
+        'max-scale': 1,
+
+        'reliability-thr': 0.7,
+        'repetability-thr': 0.7,
```

Comment:
1. I usually use underscored instead of dashes for config entries
2. Could we replace `top-k` with `max_keypoints` to be consistent with `superpoint.py`
3. `scale-f` is not very explicit, is it `scale_factor`?
4. `thr` --> `threshold`

Reply:
These variable names are set to be consistent with the original r2d2 repo. But I believe your suggestion of making it consistent with hloc is more reasonable.